frontend/package.json

```diff
@@ -44,12 +44,13 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "prettier": "^3.0.3",
     "react-scripts": "^5.0.1",
+    "typescript": "^5.2.2",
     "yalc": "1.0.0-pre.53"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject",
+    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
+    "build": "DISABLE_ESLINT_PLUGIN=true react-scripts build",
+    "eject": "DISABLE_ESLINT_PLUGIN=true react-scripts eject",
```

Comment:
Couldn't this be put on the .env file ? There is already two env vars there (`EXTEND_ESLINT=true` and
`SKIP_PREFLIGHT_CHECK=true`)

Reply:
Good point, I'll check that! :)

Reply:
I'm not sure why but for me, moving it to the `.env` file is not working..
https://github.com/facebook/create-react-app/discussions/11948

Reply:
OK, never mind then !